README.md

```diff
@@ -1,6 +1,6 @@
 # Insight
 
-Insight is a blockchain data processing tool designed to fetch, process, and store on-chain data. It provides a solution for indexing blockchain data, facilitating efficient querying of transactions, logs through a simple API.
+Insight is a blockchain data processing tool designed to fetch, process, and store on-chain data. It provides a solution for indexing blockchain data, facilitating efficient querying of blocks, transactions and logs through a simple API.
```

Comment:
you can't query blocks from insight it's only TXs and events

Reply:
yep added this before testing the API, removed now :)